spec/helpers/homepage_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+# Specs in this file have access to a helper object that includes
```

Comment:
How should this be tested. I think we should thing about this before we merge with master.